Mathlib/Data/Finsupp/Basic.lean

```diff
@@ -1676,12 +1676,13 @@ theorem distribMulActionHom_ext {f g : (α →₀ M) →+[R] N}
 /-- See note [partially-applied ext lemmas]. -/
 @[ext]
 theorem distribMulActionHom_ext' {f g : (α →₀ M) →+[R] N}
-    (h : ∀ a : α, f.comp (DistribMulActionHom.single a) = g.comp (DistribMulActionHom.single a)) :
+    (h : ∀ a : α, f.comp (DistribMulActionHom.single a) =
+      g.comp (DistribMulActionHom.single a)) :
```

Comment:
Why?

Reply:
My screen was probably too small, and it's a reflex for me to never go beyond 80char. I joined the lines.